Automattic/newspack-plugin
# Release: remove unnecessary files from the generated zip

### All Submissions:






### Changes proposed in this Pull Request:

After these changes, the generated zip will be smaller, as it won't contain unnecessary development files.

### How to test the changes in this Pull Request:

1. run `npm run release`
2. unzip the `assets/release/newspack-plugin.zip`
3. observe that it does not include `assets`, `test`, and `vendor` directories

### Other information:
